objectivec/GPBDictionary.m

```diff
@@ -596,39 +596,6 @@ void GPBDictionaryReadEntry(id mapDictionary,
 //%  NSMutableDictionary *_dictionary;
```

Comment:
You'll need to double check this by hand, I think it was Bool -> Enum that isn't generated by PDDM, so you likely have to hand edit it.  (it would be a single case, so it wasn't clear if doing macros for it was worth it.